### PR TITLE
update test runner extensions

### PR DESCRIPTION
**Pull request by ferenckiss89**
Version update of test runner fe extensions and newly builded css files.

### CODE REVIEW

File: manifest.php [hunk 1]
```diff
@@ -41,7 +41,7 @@
     'label'       => 'QTI test model',
     'description' => 'TAO QTI test implementation',
     'license'     => 'GPL-2.0',
-    'version'     => '34.3.1',
+    'version'     => '34.3.2',
```
Comment (krampstudio) [inline manifest.php]: Same remark, let's jump a feature version and enforce the dependency to `tao >= 38.5.0` and `taoTests` also

File: views/package.json [hunk 2]
```diff
@@ -8,7 +8,7 @@
   },
   "license": "GPL-2.0",
   "dependencies": {
-    "@oat-sa/tao-test-runner": "^0.2.1",
-    "@oat-sa/tao-test-runner-qti": "^0.3.2"
+    "@oat-sa/tao-test-runner": "^0.4.0",
```
Comment (krampstudio) [inline views/package.json]: let's remove all carrets and tilds for now

File: manifest.php (hunk 1 above)
Comment (ferenckiss89) [reply manifest.php]: ok

File: views/package.json (hunk 2 above)
Comment (ferenckiss89) [reply views/package.json]: yep

**Pull request merged**